src/tests/lvm/common/util_resize.go

```diff
@@ -102,23 +102,30 @@ func LvmVolumeResizeTest(decor string, engine common.OpenEbsEngine, volType comm
 	Expect(len(ftSizes)).To(BeNumerically("==", 1), "unexpected fio target sizes")
 
 	// second instance of e2e-fio, volume parameters should be the same as the 1st app instance
-	app2 := app
-	app2.Decor = app.Decor + "second-app"
+	ResizeApp2 = k8stest.FioApplication{
+		Decor:     fmt.Sprintf("%s-2", ResizeApp.Decor),
+		Loops:     2,
+		FsPercent: ResizeApp.FsPercent,
+	}
+
+	// if ResizeApp.FsType == common.BtrfsFsType {
```

Comment:
Remove the commented code

Reply:
[done](https://github.com/openebs/openebs-e2e/compare/01e8427e89706e7a7da303cb35879f77e3ab55d1..d92109e7bd8bcdec1e0dcaabcf44ae58ba75f223)